### DISCUSSION TITLE
Run Databend on AWS Lambda

### DISCUSSION THREAD

**Discussion by bohutang** [databendlabs/databend#6400] [Ideas]
**Use case**
Run Databend on AWS Lambda:
https://github.com/awslabs/aws-lambda-rust-runtime

Ref:
https://aws.amazon.com/cn/blogs/opensource/rust-runtime-for-aws-lambda/

**Comment by Xuanwo** [comment]
This issue is not actionable for now. Let's convert to discussion ideas instead.
